quipucords/fingerprinter/tests_fingerprint_task.py

```diff
@@ -995,6 +995,31 @@ def test_merge_fingerprint(self):
         self.assertIsNotNone(new_fingerprint.get('cpu_socket_count'))
         self.assertIsNotNone(new_fingerprint.get('cpu_core_count'))
 
+    def assert_all_fingerprints_have_sudo(self, metadata_dict):
+        """
+        Check if all fingerprints have 'has_sudo' set to True.
+
+        If its not True, all offending fingerprints will appear on the error log.
+        """
+        # fp <<< abbreviation for fingerprint
+        self.assertTrue(
+            all(fp_meta["has_sudo"] for fp_meta in metadata_dict.values()),
+            [
+                fp_name
+                for fp_name, fp_meta in metadata_dict.items()
+                if not fp_meta["has_sudo"]
+            ],
+        )
+        return True
+
+    def test_assert_all_fingerprints_have_sudo(self):
```

Comment:
Is this just a test to make sure the test for fingerprints having sudo is working? Sort of a test of the test?

Reply:
yep :nerd_face: 

kinda like a who watches the watchmen situation XD